setup.py

```diff
@@ -4,17 +4,17 @@
 
 setup(
     name="canonicalwebteam.search",
-    version="1.2.7",
+    version="1.3.0",
```

Comment:
Since this is a new feature, but now I made non-breaking change, I guess this is the best versioning.